src/home/NotificationBox.tsx

```diff
@@ -106,7 +106,7 @@ function useSimpleActions() {
   } = useSelector((state) => state.account)
 
   const phoneNumberVerified = useSelector(phoneNumberVerifiedSelector)
-  const numberVerifiedDecentrally = useSelector(numberVerifiedSelector)
+  const numberVerifiedDecentrally = useSelector(numberVerifiedDecentrallySelector)
```

Comment:
this is the key change, aside from the renaming

Reply:
I don't fully follow.
The same redux state is read: `state.app.numberVerified`. What's different?